### DISCUSSION TITLE
gorelease.yml not triggered

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2247] [Q&A] [answered]
Following up on https://github.com/goreleaser/goreleaser/issues/2093, which has the same symptom I'm facing -- _"gorelease.yml is never triggered"_. 

Basically I copied the `.goreleaser.yaml` and `.github/workflows/build.yml` from `goreleaser/example`, which I think is enough, right? 

Have I missed any steps?

**Comment by caarlos0** [comment]
have you created and pushed a tag?

**Comment by suntong** [comment]
Yeah, I did, https://github.com/suntong/OpenSesame/releases/tag/v1.0.2
Detailed changes are: https://github.com/suntong/OpenSesame/commits/main

**Reply by caarlos0** [reply]
it seems like the build is being triggered, but its failing way before actually running goreleaser...

https://github.com/suntong/OpenSesame/actions/runs/873304752

**Comment by caarlos0** [comment, marked as answer]
The workflow is not even starting, as mentioned in https://github.com/goreleaser/goreleaser/discussions/2247#discussioncomment-782450

Please review your workflow config.

**Comment by suntong** [comment]
Oh, thanks @caarlos0!

For anyone else landed here. the solution is --
https://github.community/t/checkout-v2-error-actions-in-this-workflow-must-within-a-repository-owned-by-organization/136315
